x/globalfee/feechecker/feechecker.go

```diff
@@ -125,106 +116,6 @@ func (fc FeeChecker) CheckTxFee(
 	return feeCoins, priority, nil
 }
 
-// IsBypassMinFeeTx checks whether tx is min fee bypassable.
-func (fc FeeChecker) IsBypassMinFeeTx(ctx sdk.Context, tx sdk.Tx) bool {
-	newCtx, _ := ctx.CacheContext()
-
-	// Check if all messages are free
-	for _, msg := range tx.GetMsgs() {
-		if !fc.IsBypassMinFeeMsg(newCtx, msg) {
-			return false
-		}
-	}
-
-	return true
-}
-
-// IsBypassMinFeeMsg checks whether msg is min fee bypassable.
-func (fc FeeChecker) IsBypassMinFeeMsg(ctx sdk.Context, msg sdk.Msg) bool {
-	switch msg := msg.(type) {
-	case *oracletypes.MsgReportData:
-		if err := checkValidMsgReport(ctx, fc.OracleKeeper, msg); err != nil {
-			return false
-		}
-	case *feedstypes.MsgSubmitSignalPrices:
-		if _, err := fc.FeedsMsgServer.SubmitSignalPrices(ctx, msg); err != nil {
-			return false
-		}
-	case *tsstypes.MsgSubmitDKGRound1:
-		if _, err := fc.TSSMsgServer.SubmitDKGRound1(ctx, msg); err != nil {
-			return false
-		}
-	case *tsstypes.MsgSubmitDKGRound2:
-		if _, err := fc.TSSMsgServer.SubmitDKGRound2(ctx, msg); err != nil {
-			return false
-		}
-	case *tsstypes.MsgConfirm:
-		if _, err := fc.TSSMsgServer.Confirm(ctx, msg); err != nil {
-			return false
-		}
-	case *tsstypes.MsgComplain:
-		if _, err := fc.TSSMsgServer.Complain(ctx, msg); err != nil {
-			return false
-		}
-	case *tsstypes.MsgSubmitDEs:
-		acc, err := sdk.AccAddressFromBech32(msg.Sender)
-		if err != nil {
```

Comment:
does this mean we don't need some keeper/msg server in feechecker anymore?

Reply:
ok